strongbox-cron/strongbox-cron-api/src/main/java/org/carlspring/strongbox/cron/jobs/CronJobsDefinitionsRegistry.java

```diff
@@ -24,7 +23,7 @@
                                              .stream()
                                              .map(ThrowingFunction.unchecked(clazz -> clazz.newInstance()
                                                                                            .getCronJobDefinition()))
-                                             .collect(ImmutableSet.toImmutableSet());
+                                             .collect(Collectors.toSet());
```

Comment:
Is there a reason for us to be changing this from immutable to mutable? Wouldn't this introduce potential concurrency/thread/ordering issues?

Reply:
AFAIK `Collectors.toSet()` will still be immutable.

Reply:
I was wrong, fixed :)